es.json

```diff
@@ -1,134 +1,134 @@
 {
-  "button.trades": "Trades",
-  "button.about": "About",
-  "button.update.apply": "Apply update",
-  "button.action.start": "Start",
+  "button.trades": "Intercambios",
+  "button.about": "Sobre",
+  "button.update.apply": "Aplicar actualización",
+  "button.action.start": "Comenzar",
   "button.action.start.hint": "⌨ Enter",
-  "button.action.init": "Start",
+  "button.action.init": "Iniciar",
   "button.action.init.hint": "⌨ Enter",
-  "button.action.stop": "Stop",
+  "button.action.stop": "Detener",
   "button.action.stop.hint": "⌨ Enter",
-  "button.action.update": "Update",
+  "button.action.update": "Actualizar",
   "button.action.update.hint": "⌨ Enter",
-  "button.mute.hint": "Notification sound",
-  "button.preview.hint": "Copy selected maps to the clipboard for /trade 159, 820",
-  "button.settings.hint": "Settings",
-  "button.preview.notify": "Copied to Clipboard",
+  "button.mute.hint": "Notificación con sonido",
+  "button.preview.hint": "Copiar mapas seleccionados al portapapeles para /trade 159, 820",  
+  "button.settings.hint": "Configuración",
+  "button.preview.notify": "Copiado al portapapeles",
   "button.notify.ok": "OK",
-  "button.notify.connect.force": "Break In",
-  "button.notify.league.ignore": "Ignore",
-  "button.notify.page.refresh": "Refresh",
-  "button.notify.loader.reload": "Reload",
-  "update.text": "Update available",
-  "connection.init": "Connecting...",
-  "board.tier": "Tier {{number}}",
-  "board.unique": "Unique",
-  "board.fragments": "Fragments",
-  "board.info.have": "You have",
-  "board.info.want": "You want",
-  "board.tier.selected": "tiers selected",
-  "board.label.main": "tier-to-tier exchange",
-  "board.map.type.regular": "Regular",
+  "button.notify.connect.force": "Forzar",
+  "button.notify.league.ignore": "Ignorar",
+  "button.notify.page.refresh": "Refrescar",
+  "button.notify.loader.reload": "Recargar",
+  "update.text": "Actualización disponible",
+  "connection.init": "Conectando...",
+  "board.tier": "Grado {{number}}", 
+  "board.unique": "Único",
+  "board.fragments": "Fragmentos",
+  "board.info.have": "Tienes",
+  "board.info.want": "Quieres",
+  "board.tier.selected": "grados seleccionados",
+  "board.label.main": "intercambio de grado-a-grado",
+  "board.map.type.regular": "Mapa regular",
```

Comment:
Is it possible to translate the fields "board.map.type.regular", "board.map.type.shaper", "board.map.type.elder" in one word? Suddenly, we don't have much space there to use it.

Reply:
What would be the context of the word Regular?
For both "board.map.type.shaper" and "board.map.type.elder" we could use Shaper and Elder since most of spanish speakers play the game in english.

Reply:
PoE has 3 types of maps. We call them "Regular" for maps without any modification, "Shaped" with applied Shaper's orb, and "Elder" for maps with applied Elder's orb

Reply:
Yup! Thanks